### PR TITLE
structure refactoring

### PR DESCRIPTION
**Pull request by khemrajrathore**
Refactoring the RuleInfo remains, as new ruleStructures will be needed for JS

### CODE REVIEW

**Comment by pandurangpatil** [comment]
@khemrajrathore have you tested the one or two repos with this code refactor?

File: src/main/scala/ai/privado/semantic/Language.scala [hunk 1]
```diff
@@ -22,18 +22,10 @@
 
 package ai.privado.semantic
```
Comment (pandurangpatil) [inline src/main/scala/ai/privado/semantic/Language.scala:24]: @khemrajrathore did we create a new Language.scala as a subpackage inside java package?

Comment (khemrajrathore) [reply src/main/scala/ai/privado/semantic/Language.scala:24]: yes, we had this earlier outside, now we have segregated into 2, generic ones outside, lang specific inside java

**Comment by pandurangpatil** [comment]
@ojaswa1942 If you find the changes good, go ahead and merge the changes.

**Review by ojaswa1942 — APPROVED** [review]

**Pull request merged**